tests.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+const {app} = require('electron')
+
+app.on("ready", function() {
+    require("./node_modules/gpii-windows/tests/UnitTests.js");
```

Comment:
This hard-coding to a path relative to node_modules will be unstable between versions of npm.
Instead, I suggest something like
var fluid = require("infusion");
require("gpii-windows");
fluid.require("%gpii-windows/tests/UnitTests.js");